src/main/java/com/project/moflis/controller/Controller.java

```diff
@@ -1,15 +1,15 @@
 package com.project.moflis.controller;
 
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 import org.springframework.web.bind.annotation.GetMapping;
 
 @RestController
 public class Controller {
 
-    @GetMapping("hi")
-    public String getMethodName() {
-        return "하이";
+    @RequestMapping("hi")
+    public String hello() {
+        return "hi";
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 8, expected level should be 4.
<!-- __reviewdog__:Cg9iYjllZGY5NTA0NzY5MWQSCmNoZWNrc3R5bGU= -->